pkg/executor/bacalhau/bacalhau.go

```diff
@@ -69,6 +79,26 @@ func (executor *BacalhauExecutor) Id() (string, error) {
 	return idResult.ID, nil
 }
 
+// Checks that Bacalhau is installed, correctly configured, and available
+func (executor *BacalhauExecutor) IsAvailable() (bool, error) {
+	isAlive, err := executor.bacalhauClient.Alive()
+	if !isAlive || err != nil {
+		return false, errors.New("Bacalhau is not currently available")
```

Comment:
```suggestion
		return false, errors.New("Bacalhau is not currently available. Please ensure that Bacalhau is running, then try again.")
```
We made want to add a suggestion for which course of action to take.